cli/template/base/tsconfig.json

```diff
@@ -21,6 +21,13 @@
       "~/*": ["./src/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "**/*.cjs", "**/*.mjs"],
+  "include": [
+    ".eslintrc.cjs",
```

Comment:
Added to ensure TypeScript checks the file (files in project root are not matched by `**/*.cjs`).

Reply:
fyi: files in root are matched but files starting in `.` isn't.

anywho this is needed nevertheless

Reply:
Ah, TIL!